authority/authority_test.go

```diff
@@ -113,24 +114,24 @@ func TestAuthorityNew(t *testing.T) {
 					assert.True(t, auth.initOnce)
 					assert.NotNil(t, auth.intermediateIdentity)
 					for _, p := range tc.config.AuthorityConfig.Provisioners {
-						_p, ok := auth.provisionerIDIndex.Load(p.ID())
+						_p, ok := auth.provisioners.Load(p.GetID())
 						assert.True(t, ok)
 						assert.Equals(t, p, _p)
-						if len(p.EncryptedKey) > 0 {
-							key, ok := auth.encryptedKeyIndex.Load(p.Key.KeyID)
+						if kid, encryptedKey, ok := p.GetEncryptedKey(); ok {
+							key, ok := auth.provisioners.LoadEncryptedKey(kid)
 							assert.True(t, ok)
-							assert.Equals(t, p.EncryptedKey, key)
+							assert.Equals(t, encryptedKey, key)
 						}
 					}
 					// sanity check
-					_, ok = auth.provisionerIDIndex.Load("fooo")
+					_, ok = auth.provisioners.Load("fooo")
 					assert.False(t, ok)
 
-					assert.Equals(t, auth.audiences, []string{
-						"step-certificate-authority",
-						"https://127.0.0.1/sign",
-						"https://127.0.0.1/1.0/sign",
-					})
+					// assert.Equals(t, auth.audiences, []string{
```

Comment:
dead (test) code?

Reply:
Yes, removing it, authority doesn't have the audiences stored, they will be different depending on the provisioner type.